packages/terra-hyperlink/tests/wdio/hyperlink-spec.js

```diff
@@ -0,0 +1,120 @@
+      'color-contrast': { enabled: false },
+    };
+
+    Terra.should.beAccessible({ viewports, rules });
+    Terra.should.matchScreenshot({ viewports });
+    Terra.should.themeEachCustomProperty({
+      '--terra-link-disabled-color': 'red',
+    });
+  });
+
+  describe('Audio', () => {
+    before(() => browser.url('/#/raw/tests/terra-hyperlink/hyperlink/audio-hyperlink'));
+
+    Terra.should.beAccessible({ viewports });
+    Terra.should.matchScreenshot({ viewports });
+    Terra.should.themeEachCustomProperty({
+      '--terra-link-audio-icon': 'linear-gradient(red, orange)',
+    });
+  });
+
+  describe('Document', () => {
+    before(() => browser.url('/#/raw/tests/terra-hyperlink/hyperlink/document-hyperlink'));
+
+    Terra.should.beAccessible({ viewports });
+    Terra.should.matchScreenshot({ viewports });
+    Terra.should.themeEachCustomProperty({
+      '--terra-link-document-icon': 'linear-gradient(red, orange)',
+    });
+  });
+
+  describe('External', () => {
+    before(() => browser.url('/#/raw/tests/terra-hyperlink/hyperlink/external-hyperlink'));
+
+    Terra.should.beAccessible({ viewports });
+    Terra.should.matchScreenshot({ viewports });
+    Terra.should.themeEachCustomProperty({
+      '--terra-link-external-icon': 'linear-gradient(red, orange)',
+    });
+  });
+
+  describe('Image', () => {
+    before(() => browser.url('/#/raw/tests/terra-hyperlink/hyperlink/image-hyperlink'));
+
+    Terra.should.beAccessible({ viewports });
+    Terra.should.matchScreenshot({ viewports });
+    Terra.should.themeEachCustomProperty({
+      '--terra-link-image-icon': 'linear-gradient(red, orange)',
+    });
+  });
+
+  describe('Video', () => {
+    before(() => browser.url('/#/raw/tests/terra-hyperlink/hyperlink/video-hyperlink'));
+
+    Terra.should.beAccessible({ viewports });
+    Terra.should.matchScreenshot({ viewports });
+    Terra.should.themeEachCustomProperty({
+      '--terra-link-video-icon': 'linear-gradient(red, orange)',
+    });
+  });
+
+  describe('Icon Non Wrapping', () => {
+    before(() => browser.url('/#/raw/tests/terra-hyperlink/hyperlink/icon-non-wrapping-hyperlink'));
+
+    Terra.should.beAccessible({ viewports });
+    Terra.should.matchScreenshot({ viewports });
+  });
+
+  describe('Scaled', () => {
+    before(() => browser.url('/#/raw/tests/terra-hyperlink/hyperlink/scaled-hyperlink'));
+
+    Terra.should.beAccessible({ viewports });
+    Terra.should.matchScreenshot({ viewports });
+  });
+
+  describe('States', () => {
+    before(() => browser.url('/#/raw/tests/terra-hyperlink/hyperlink/states-hyperlink'));
+
+    Terra.should.beAccessible({ viewports });
+    Terra.should.matchScreenshot({ viewports });
+    Terra.should.themeCombinationOfCustomProperties({
+      testName: 'themed',
+      properties: {
+        '--terra-link-color': 'red',
+        '--terra-link-underline': 'underline',
+        '--terra-link-visted-color': 'orange',
+        '--terra-link-hover-color': 'yellow',
```

Comment:
This theme test needs broken into two different ones, one for hover state and one for focus state to fully capture the style differences

Reply:
The test URL this uses has individual links for each state so its already broken up where each state can be captured with its unique styles

Reply:
Oh thats works then! 😄